packages/client/hmi-client/src/services/goLLM.ts

```diff
@@ -54,7 +54,33 @@ export async function configureModel(documentId: string, modelId: string): Promi
 			}
 		});
 	} catch (err) {
-		logger.error(`An issue occured while exctracting a model configuration. ${err}`);
+		logger.error(`An issue occured while exctracting a model configuration from document. ${err}`);
+	}
+}
+
+export async function configureModelFromDatasets(modelId: string, datasetIds: string[]) {
+	try {
+		// FIXME: Using first dataset for now...
+		const response = await API.post<TaskResponse>('/gollm/configure-from-dataset', null, {
+			params: {
+				'model-id': modelId,
+				'dataset-ids': datasetIds[0]
```

Comment:
Try the following before changing from params to body
```suggestion
				'dataset-ids': datasetIds.join()
```

Reply:
this worked 👍